server/commands/battle.py

```diff
@@ -0,0 +1,736 @@
+                                        effect="statdown",
+                                    )
+                                if "spddown" in move.effect:
+                                    target.battle.spd = (
+                                        target.battle.spd / area.battle_bonus_malus
+                                    )
+                                    battle_send_ic(
+                                        target,
+                                        msg=f"The special defense of ~{target.battle.fighter}~ goes down",
+                                        effect="statdown",
+                                    )
+                                if "spedown" in move.effect:
+                                    target.battle.spe = (
+                                        target.battle.spe / area.battle_bonus_malus
+                                    )
+                                    battle_send_ic(
+                                        target,
+                                        msg=f"The speed of ~{target.battle.fighter}~ goes down",
+                                        effect="statdown",
+                                    )
+                                if "poison" in move.effect:
+                                    if target.battle.status is None:
+                                        target.battle.status = "poison"
+                                        battle_send_ic(
+                                            target,
+                                            msg=f"~{target.battle.fighter}~ is affected by poisoning",
+                                            effect="poison",
+                                            shake=1,
+                                        )
+                                if "paralysis" in move.effect:
+                                    if target.battle.status is None:
+                                        target.battle.status = "paralysis"
+                                        battle_send_ic(
+                                            target,
+                                            msg=f"~{target.battle.fighter}~ is affected by paralysis",
+                                            effect="paralysis",
+                                            shake=1,
+                                        )
+                                if target.battle.hp <= 0:
+                                    battle_send_ic(
+                                        target,
+                                        msg=f"~{target.battle.fighter}~ dies...",
+                                        offset=100,
+                                    )
+                            else:
+                                if len(targets) == 1:
+                                    battle_send_ic(
+                                        client, msg="but the target is already died"
```

Comment:
minor spelling mistake *evaporates you*
should be "but the target is already dead"